src/libraries/System.Linq.Queryable/ref/System.Linq.Queryable.cs

```diff
@@ -37,25 +37,18 @@ public partial class EnumerableQuery<T> : System.Linq.EnumerableQuery, System.Co
     }
     public static partial class Queryable
     {
-        [System.Diagnostics.CodeAnalysis.RequiresDynamicCode("Enumerating collections as IQueryable can require creating new generic types or methods, which requires creating code at runtime. This may not work when AOT compiling.")]
```

Comment:
Are these because we generally don't include RequiresUnreferencedCodeAttribute in ref files?

Reply:
It's because @roji removed them from the implementation in https://github.com/dotnet/runtime/commit/76e2be8f6a8798d15817ab31a5b2ccd594f4bc0a